### ISSUE TITLE
Some ideas

### ISSUE THREAD

**Issue by Firesoul7** [PhelanBavaria/ancienttimeline#49]
I had a few ideas today regarding improvements to how I think AT should work. My vision for AT is a world where it is genuinely difficult to hold onto land, where mistakes can be extremely costly and for small nations can result in them disappearing from the face of the Earth. I've already raised some ideas regarding events to reduce development, and since I've got some holidays coming up I might be able to do some work on this. In addition, I believe that there should be occasional events that spread cultures both within and without the borders of medium and large sized countries, I'm thinking anything over 15 provinces. Right now it's way too easy to just create a Roman lake, I think that if you're playing as say Rome or Macedon or whoever that non-primary cultures within their borders, aswell as cultures adjacent to their borders, should slowly spread through their territory by event, simulating the migration of members of these cultures into and throughout these nations. Next, I believe that a number of non-historical nations should be created for each major culture. My vision of the world at this time is one of lots of small nations struggling to hold onto their land, and many of them failing to do so. The result of this would ultimately be an empty world, which is not desirable, so every so often new nations would appear randomly in empty provinces. The nation that would appear would be chosen from a number of big lists of countries both historical and unhistorical that are not currently active, with a different list for all the major cultures. This would further tie with the colonisation mechanics. A nation that creates an overseas colony would immediately lose control of that colony if it is successful, with the colony being transferred to a member of one of these lists. Of course a colony is a massive investment (or will be) that can potentially sink a nation, so the new country that would appear would be a vassal of the coloniser. Having a vassal of the same religion and culture would be a big asset in this world of mostly small OPMs. Finally, the elimination of all wastelands, which of course will be the last thing to be done on this list. What does everyone think of my ideas?

**Comment by fangwolf4** [comment]
in my opinion,why not just bring back the wastelands,that way colonization is harder and the world is less empty in a sense

**Comment by Firesoul7** [comment]
There are already wastelands, and I think that they're immensely stupid. A wasteland is a magic barrier ingame, even the very best supply train in the world and an investment of thousands of tons of gold will not allow a man to take a single step into one. If Rome at its height decided to invest all its energy and resources into colonising a part of Siberia, it could have done it. Ancient humans colonised Kamchatka, and crossed from there to America. Certainly a Siberian colonisation venture would cripple Rome and give basically no reward, but it was a possibility. Right now basically the whole world is just wastelands, and I loathe it with a passion. My ideas allow us to get rid of those wastelands without allowing enormous blobs to rule the Earth.

**Comment by qweytr** [comment]
I agree that blobbing and spreading culture must be made very difficult, but limiting expansion too much might result in boring gameplay. Wastelands are necessary for now and removing some of them can be discussed later if blobbing can be prevented well enough.

**Comment by Firesoul7** [comment]
It's true that for example the current system in ET can get a bit boring.
It's necessary, but getting the same event over and over is a pain. The
idea is that under this system the player has to think tactically in order
to hold on. The ET system has no risk, it just stops high development
provinces, my system if it can be got working means that the player is
using actual tactics (and has to) just to stay in the game, which I think
is a lot better.

**Comment by qweytr** [comment]
What you are describing sounds good. My only concern is whether it's possible to do it like that in practice, but if you can then go ahead.
